handlers/custom_reminder_handler.py

```
@@ -0,0 +1,374 @@
+        text = (update.message.text or "").strip()
+        context.user_data.pop(AWAITING_TEXT, None)
+
+        if not text:
+            await update.message.reply_text(f"{config.EMOJIS['error']} התזכורת ריקה, לא נשמרה.")
+            return True
+        if len(text) > MAX_TEXT_LENGTH:
+            text = text[:MAX_TEXT_LENGTH]
+
+        context.user_data[DRAFT] = {"text": text}
+        await update.message.reply_text(
+            f'{config.EMOJIS["reminder"]} <b>{text}</b>\n\nמתי להזכיר?',
```

Ruff (0.16.1)

[warning] String contains ambiguous `י` (HEBREW LETTER YOD). Did you mean ``` (GRAVE ACCENT)?

(RUF001)